test/test_tf_frontend_router.py

```diff
@@ -43,16 +43,20 @@ def pattern(placeholder, open_curly, close_curly, text):
 
 class TestTFFrontendRouter(unittest.TestCase):
 
-    def setUp(self):
-        self.workdir = tempfile.mkdtemp()
-        self.base_path = os.getcwd()
-        self.module_path = os.path.join(os.getcwd(), 'test', 'infra')
+    @classmethod
```

Comment:
Would you be able to elaborate, why this has changed, please?

Reply:
The modules were being downloaded before each test and it was taking 10 minutes to run all the tests. This just does it before all the tests.